src/routes/alerts.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const fetch = require('node-fetch');
+
+const router = express.Router();
+require('dotenv').config();
+
+
+router.get('/alerts', (req, res) => {
+  const key = process.env.FIVE_ONE_ONE_API_KEY
+     fetch(`http://api.511.org/transit/servicealerts?api_key=${key}&format=json`)
+     .then(response => {
+       return response.text();
```

Comment:
@Sanderson239 I notice on the next line you refer to `json` but here are using the `.text()` method. I think if you use the `json()` method here, you will not actually need to parse the JSON below.